config.toml

```diff
@@ -88,21 +88,15 @@ breadcrumb_disable = false
 sidebar_search_disable = false
 
 [[params.versions]]
-  version = "v0.11 (latest)"
-  url = "https://release-v0-11.docs.openservicemesh.io/"
+  version = "v1.1 (latest)"
+  url = "https://release-v1-1.docs.openservicemesh.io/"
 [[params.versions]]
-  version = "v0.10"
-  url = "https://release-v0-10.docs.openservicemesh.io/"
-[[params.versions]]
-  version = "v0.9"
-  url = "https://release-v0-9.docs.openservicemesh.io/"
-[[params.versions]]
-  version = "v0.8"
-  url = "https://release-v0-8.docs.openservicemesh.io/"
```

Comment:
Even though the upstream project might not support the version anymore, users could still be using an older version. We have seen this happen in the past.

There is little overhead in preserving older versions in the dropdown along with a banner stating it has been archived. This way, users on an older version still have documentation to refer to.

Reply:
I'm fine with what @shashankram says, if we all agree. As long as we very clearly say that we don't support this version, then I don't think it does any harm. My original justification was the fear that `documented == supported`. But if we can decouple that then I don't think there is a big issue.

Reply:
Keeping documentation online for all possible old versions carries its own risks (of people believing that because they found docs, it's okay to use something very old). The Kubernetes project itself its docs for several unsupported versions, but not _all_ versions: https://kubernetes.io/docs/home/supported-doc-versions/

It's probably a good idea to clearly document whatever you choose to do.